xbmc/cores/VideoPlayer/test/edl/TestEdl.cpp

```diff
@@ -274,23 +288,27 @@ TEST_F(TestEdl, TestComSkipVersion2EDL)
   EXPECT_EQ(found, true);
   // fps is obtained from the file as it always takes precedence (note we supplied 0 above),
   // the EDL file has the value of 2500 for fps. kodi converts this to 25 fps by dividing by a factor of 100
-  const float fpsInEdlFile = 2500 / 100;
+  const double fpsInEdlFile = 2500 / 100;
```

Comment:
Why this change?

Reply:
Because it could lead to wrong approximations (floating point) when doing the later comparisons against seconds with decimal places (the internal value is correct but the conversion from tests in seconds can lead the test to fail in the CI). We internally store milliseconds and all the comparisons are done in milliseconds. Some tests are written comparing to seconds just for increased readability.